### ISSUE TITLE
Accessibility: screen reader repeat timestamp multiple times

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3075]
## Version

`4.8.0`

## Describe the bug

Some usability on screen reader:

- Screen reader repeat timestamp multiple times
- There is no clear association between the activity text, and its attachment

## Steps to reproduce

1. Turn on Windows Narrator with either Chrome 80 or Edge 80+
1. Type "Hello" to the bot
1. Bot to send a hero card with buttons, in an activity with text

## Expected behavior

- Activity text should only be narrated once
- When narrating the hero card, it should be "Activity text region. Yes button."

## Additional context

Currently, there is a [bug on NVDA 2019.3.1](https://github.com/nvaccess/nvda/issues/10866) (latest as of now) not working with Chrome 80 (latest as of now) with `aria-live="polite"`. Please use Windows Narrator.

[Bug]

**Comment by feerrenrut** [comment]
This is most likely a bug in Chrome / Edge. Please use Firefox.

**Comment by compulim** [comment]
It's a bug on Chromium, https://bugs.chromium.org/p/chromium/issues/detail?id=1067257.
